scripts/serverless.js

```diff
@@ -37,8 +37,25 @@ const processSpanPromise = (async () => {
     const uuid = require('uuid');
     const Serverless = require('../lib/Serverless');
     const resolveConfigurationPath = require('../lib/cli/resolve-configuration-path');
+    const isHelpRequest = require('../lib/cli/is-help-request');
+    const readConfiguration = require('../lib/configuration/read');
 
-    serverless = new Serverless({ configurationPath: await resolveConfigurationPath() });
+    const configurationPath = await resolveConfigurationPath();
+    const configuration = configurationPath
+      ? await (async () => {
+          try {
+            return await readConfiguration(configurationPath);
+          } catch (error) {
+            if (isHelpRequest()) return null;
```

Comment:
Is there a reason to check for `isHelpRequest` only during errored out `readConfiguration` call? Maybe there's not need for resolving path and loading configuration in such case and we could introduce that check before `resolveConfigurationPath` ?

Reply:
Good question. Reason for that is that if there are plugins in a _service_ then they influence _help_ output (commands they introduce are listed in help)

So we also want to parse config if it's a _help_ request, but failure of parse should not prevent display of help

Reply:
Understood, thanks for the explanation :bow:

Reply:
I've added comment explaining that.